Cesium.CodeGen/Ir/Types/PointerType.cs

```diff
@@ -50,4 +50,8 @@ public IExpression GetSizeInBytesExpression(TargetArchitectureSet arch)
 
         return new SizeOfOperatorExpression(this);
     }
+    public override int GetHashCode()
+    {
+        return base.GetHashCode();
+    }
```

Comment:
Oh, is it really necessary? :)

Reply:
This is requred to break cycle in struct GetHashCode when struct has pointer of itself's as member type.

Reply:
Oh boy

Reply:
Perhaps let's break it in the struct type then somehow? It looks weird to break the `PointerType` because of that. In its current state, it is not very useful as a key in a hash table.